Makefile

```diff
@@ -10,6 +10,9 @@ STRIP = strip
 .PHONY: all
 all: immu immud immugw
 
+.PHONE: rebuild
```

Comment:
Isn't this a typo? I suppose it should be `.PHONY`

Reply:
fixed